judo-ui-react/src/main/resources/actor/src/components/widgets/Tags.tsx.hbs

```diff
@@ -1,19 +1,21 @@
 {{> fragment.header.hbs }}
 
+import { Button } from '@mui/material';
```

Comment:
Please re-write this to `import Button from '@mui/material/Button';`, because it's the optimized version for dev-mode.